index1.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link url="../Academy/Lato/" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i" rel="stylesheet">
+    <link href="../Academy/style1.css" rel="stylesheet">
+
+    <title>Landing Page</title>
+
+</head>
+
+<body>
+    <div class="container">
+        <img class="coming-soon" src="../Academy/assets/coming-soon-ribbon.png" alt="coming soon" />
+
+        <header class="clearfix">
+            <img class="header_image" src="../Academy/assets/undraw_developer_activity_bv83.svg" alt="undraw_developer" />
+            <h1>CODE ACADEMY</h1>
```

Comment:
Zdecydowanie lepiej napisać normalnie Code Academy, a wielkie litery zrobić za pomocą text-transform: uppercase.

Reply:
ale wtedy trzeba skorelować dodatkowy styl z danym elementem, po co?

Reply:
Wydaje mi się, że to nie kwestia rozpatrywania tego jako dodatkowego stylowania, czy rozgraniczania co jest wydajniejsze, a co nie. Ale zastosowanie text-transform jest wygodne i estetyczniejsze, imho. O ile w tym przypadku to nie ma znaczenia, bo jest to małe, jest to statyczne, to chyba chodzi o to żeby się uczyć dobrych praktyk, które przeniesiemy do większych i bardziej zaawansowanych projektów. Nie wydaje mi się żebyś na jakimś dużym projekcie walił nagłówki capslockiem w HTMLu.